crates/core_arch/src/aarch64/tme.rs

```diff
@@ -71,7 +71,7 @@ pub const _TMFAILURE_TRIVIAL: u64 = 1 << 24;
 /// [ARM TME Intrinsics](https://developer.arm.com/docs/101028/0010/transactional-memory-extension-tme-intrinsics).
 #[inline]
 #[target_feature(enable = "tme")]
-#[cfg_attr(test, assert_instr(tstart))]
+#[cfg_attr(test, assert_instr(nop))]
```

Comment:
Why were these changed to nop? This is incorrect.

Reply:
The `assert_instr` of these instructions failed on my local aarch64. But since these instructions are not included in this stabilization, I think we can keep them unchanged.

Reply:
It might have failed because the binutils version on your system is too old and objdump is not able to disassemble the instruction. We use a recent version of ubuntu in the docker image used in CI for this reason.